explainer.md

```diff
@@ -54,18 +55,18 @@ const image = await clipboardItem.getType('image/png');
 
 ## Pickling Write
 Pickled formats will be written when the site tries to use a format that:
-1. Is included in a `{direct: ['format1', 'format2']}` list of formats for the `ClipboardItem`’s `ClipboardItemOption`. 
+1. Is included in a `{unsanitize: ['format1', 'format2']}` list of formats for the `ClipboardItem`’s `ClipboardItemOption`. 
 2. Regardless of whether a sanitized format is also written.
 
-For example, writing the format `"text/plain"`, which is recognized by the Clipboard API as a sanitized format, will write this format to the clipboard as usual. On the other hand, writing a format `"text/custom-format"`, while also providing a `{direct: ['text/custom-format']}` list will result in a pickled format corresponding to `"text/custom-format"` to be written to the clipboard. And writing “text/plain” with a `{direct: ['text/plain']}` list will write both a sanitized and pickled payload. This API change allows a potential related pickling and sanitized format to coexist on the clipboard at the same time, which allows sites to interoperate with the more widely used sanitized format, while also getting the increased fidelity of the unmodified pickled format.
+For example, writing the format `"text/plain"`, which is recognized by the Clipboard API as a sanitized format, will write this format to the clipboard as usual. On the other hand, writing a format `"text/custom-format"`, while also providing a `{unsanitize: ['text/custom-format']}` list will result in a pickled format corresponding to `"text/custom-format"` to be written to the clipboard. And writing “text/plain” with a `{unsanitize: ['text/plain']}` list will write both a sanitized and pickled payload. This API change allows a potential related pickling and sanitized format to coexist on the clipboard at the same time, which allows sites to interoperate with the more widely used sanitized format, while also getting the increased fidelity of the unmodified pickled format.
```

Comment:
optional nit: For a tiny bit more info, we could say "On the other hand, writing a format text/custom-format, which is not recognized by the Clipboard API as a sanitized format, will result in a pickled format [...]". Not sure if that's too wordy though

Reply:
That sounds good. Should we also include the `{unsanitize: ['text/custom-format']}` part as that is required for pickling write (even for custom formats)?

Reply:
What do you mean by "the `{unsanitized: ['text/custom-format']}` part"?

Reply:
So if we just provide `"text/custom-format"` in the `ClipboardItem` without `{unsanitize: ['text/custom-format']}`, then it shouldn't write any formats because `unsanitize` option is required for any format write in unsanitized form right?

Reply:
Ah yes, providing an unsanitized format without the unsanitized flag shouldn't write any formats. Thanks for clarifying

Reply:
Changed it to: On the other hand, writing a format `"text/custom-format"` with a `{unsanitize: ['text/custom-format']}` list, which is not recognized by the Clipboard API as a sanitized format, will result in a pickled format corresponding to `"text/custom-format"` to be written to the clipboard. And writing “text/plain” with a `{unsanitized: ['text/plain']}` list will write both a sanitized and pickled payload.